Doc/whatsnew/2.3.rst

```diff
@@ -672,7 +672,7 @@ with older versions of the Distutils::
    from distutils import core
    kw = {'name': "Quixote",
          'version': "0.5.1",
-         'description': "A highly Pythonic Web application framework",
+         'description': "A highly Pythonic web application framework",
```

Comment:
It is a description of an external package. https://pypi.org/project/Quixote/
"Web" is used in title case there.